decentraland/screens/ListLandForSaleScreen.js

```diff
@@ -99,7 +101,8 @@ export class ListLandForSaleScreen extends React.Component {
 
     // Note: Conversion to USD will be implemented on v0.2.0
     const manaPerUsd = 30;
-    const priceMana = Number(`${priceInWei}`) / 1e18;
+    // Get mana prince using string to avoid imprecise rounded (i.e.: 57999.99999999999)
+    const priceMana = `${priceInWei}`.substring(0, `${priceInWei}`.length - 18);
```

Comment:
I worry about this many operations in one line leading to hard-to-interpret errors in stack traces

Reply:
Ok, I'll improve that.

Reply:
Done

Reply:
Thanks! Resolving this.